papaparse.js

```diff
@@ -1252,9 +1253,12 @@
 			};
 		}
 
-		function guessLineEndings(input)
+		function guessLineEndings(input, quoteChar)
 		{
 			input = input.substr(0, 1024 * 1024);	// max length 1 MB
+			// Replace all the text inside quotes
+			var re = new RegExp(quoteChar + '([^]*?)' + quoteChar, 'gm');
```

Comment:
The `quoteChar` here should be run through a regex escape. It could cause major issues if the user provided a character like `.` as the quote char.

This function from the mozilla docs should be good if there isn't already one available in this repo. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
```js
function escapeRegExp(string) {
  return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
}
```